### PR TITLE
Enhance config https

### PR DESCRIPTION
**Pull request by ChristianMayer**

### CODE REVIEW

File: source/resource/demo/visu_config_gauge.xml [hunk 1]
```diff
@@ -132,7 +132,7 @@
           <layout colspan="1.3"/>
           <address transform="DPT:16" mode="read" variant="setValue">12/7/95</address>
         </gauge>
-        <switch mapping="Text_Scroll" styling="- not set - (undefined)" on_value="CometVisu Geil" off_value="AUS" bind_click_to_widget="true">
+        <switch mapping="Text_Scroll" on_value="CometVisu Geil" off_value="AUS" bind_click_to_widget="true">
```
Comment (peuter) [inline source/resource/demo/visu_config_gauge.xml]: What does the styling setting mean, is this on purpose?

Comment (ChristianMayer) [reply source/resource/demo/visu_config_gauge.xml]: Nothing. It's a left over when the Editor is used "wrongly". (Actually it's an editor bug in my point of view)

Comment (peuter) [reply source/resource/demo/visu_config_gauge.xml]: Shouldn't we remove that from the demo + example configs then?

Comment (ChristianMayer) [reply source/resource/demo/visu_config_gauge.xml]: Yes - and this PR is doing that. Or did I miss any?

Comment (peuter) [reply source/resource/demo/visu_config_gauge.xml]: You're right. Seems that am I not able to read pull requests anymore ;-)

**Pull request merged**